app/recipe/serializers.py

```diff
@@ -0,0 +1,38 @@
+from rest_framework import serializers
+
+from core.models import Ingredient, Recipe
+
+
+class IngredientSerializer(serializers.ModelSerializer):
+    """Serializer for ingredient objects"""
+
+    class Meta:
+        model = Ingredient
+        fields = ('name',)
+        read_only_fields = ('id',)
+
+
+class RecipeSerializer(serializers.ModelSerializer):
+    """Serializer for recipe objects"""
+    ingredients = IngredientSerializer(many=True)
+
+    def create(self, validated_data):
+        ingredients = validated_data.pop('ingredients')
```

Comment:
Why don't you use the model's `create` method?
https://github.com/clicktravel-ruthmills/tk-django-recipe-exercise/pull/1/files#diff-8b91c5949f356665ee0f9636ea4d2144065b368bba6876f73d1a53879c61d92dR9

Reply:
I've now removed the `create` method from the model and kept the one in the serializer.